src/resources/nfts/index.ts

```diff
@@ -86,6 +96,6 @@ export function useLegacyNFTs({ address }: { address: string }) {
   return {
     data: nfts,
     error,
-    isFetching,
+    isInitialLoading: !nfts.length && isFetching,
```

Comment:
Not a blocker, but kinda odd naming imo. I think RQ might give us what we need with `isLoading`? https://tanstack.com/query/v4/docs/react/reference/useQuery

Reply:
Like I kinda think these hooks should _only_ return props that come from RQ, if possible. Keep them as close to RQ as possible, unless we can't.

Reply:
isLoading is always false, for some reason. there's no builtin prop that does what we want here. i chose this name bc useQuery actually has a prop of the same name that does the same thing: "is true whenever the first fetch for a query is in-flight". it doesn't exist on useInfiniteQuery though. I'm open to another name if you have an idea!

Reply:
Nope all good!